apps/scheduler/tests/test_runs.py

```
@@ -0,0 +1,197 @@
+    def test_redacts_basic_credentials(self) -> None:
+        self.assertIn(
+            'Basic [redacted]', runs.scrub('Basic dXNlcjpwYXNzd29yZA==')
+        )
+
+    def test_redacts_api_keys(self) -> None:
+        scrubbed = runs.scrub('key=ik_abcdefgh12345678 rest')
+        self.assertNotIn('ik_abcdefgh', scrubbed)
+        self.assertIn('rest', scrubbed)
+
+    def test_redacts_jwts(self) -> None:
+        token = 'eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhIn0.c2lnbmF0dXJl'
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "token"

(S105)